lib/generators/sufia/cached_stats_generator.rb

```diff
@@ -16,7 +16,8 @@ def banner
   def copy_migrations
     [
       'create_file_view_stats.rb',
-      'create_file_download_stats.rb'
+      'create_file_download_stats.rb',
+      'create_work_view_stats.rb'
```

Comment:
Since the URLs for files/works are changing, do we need to keep the sufia 6 statistics in their own table?